iib/workers/tasks/build.py

```diff
@@ -802,25 +807,31 @@ def handle_add_request(
 
         arches = prebuild_info['arches']
         if deprecation_bundles:
-            if is_fbc:
-                err_msg = 'File-Based catalog image type is not supported for deprecation yet.'
-                log.error(err_msg)
-                raise IIBError(err_msg)
             # opm can only deprecate a bundle image on an existing index image. Build and
             # push a temporary index image to satisfy this requirement. Any arch will do.
             arch = sorted(arches)[0]
             log.info('Building a temporary index image to satisfy the deprecation requirement')
             _build_image(temp_dir, 'index.Dockerfile', request_id, arch)
             intermediate_image_name = _get_local_pull_spec(request_id, arch, include_transport=True)
-            deprecate_bundles(
-                deprecation_bundles,
-                temp_dir,
-                prebuild_info['binary_image'],
-                intermediate_image_name,
-                overwrite_from_index_token,
-                # Use podman so opm can find the image locally
-                container_tool='podman',
-            )
+            if is_fbc:
+                deprecate_bundles_fbc(
```

Comment:
Do we need log message suggesting FBC image processing here?

Reply:
I think we already have this info present in log file from line 756.